core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/debug/BreakpointManagerImpl.java

```diff
@@ -120,9 +115,9 @@ public void changeBreakpointState(final int lineNumber) {
 
         final VirtualFile activeFile = editor.getEditorInput().getFile();
 
-        List<Breakpoint> breakpointsForPath = this.breakpoints.get(activeFile.getPath());
-        if (breakpointsForPath != null) {
-            for (final Breakpoint breakpoint : breakpointsForPath) {
+        List<Breakpoint> pathBreakpoints = breakpoints.get(activeFile.getPath());
+        if (pathBreakpoints != null) {
```

Comment:
I think it will be good to add checking on emptiness, it will avoid calling iterator internally.